### PR TITLE
Update event_listeners.js 'Missing credentials in config, try to run \'AWS_SDK_LOAD_CONFIG=1 npm start\''

### PR DESCRIPTION
**Pull request by odykyi**
'Missing credentials in config, try to run \'AWS_SDK_LOAD_CONFIG=1 npm start\''

### CODE REVIEW

**Comment by odykyi** [comment]
@lsegal  @konklone @corymsmith @ericb  Please review
Thanks!

File: lib/event_listeners.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ AWS.EventListeners = {
       req.service.config.getCredentials(function(err) {
         if (err) {
           req.response.error = AWS.util.error(err,
-            {code: 'CredentialsError', message: 'Missing credentials in config'});
+            {code: 'CredentialsError', message: 'Missing credentials in config, try to run \'AWS_SDK_LOAD_CONFIG=1 npm start\''});
```
Comment (srchase) [inline lib/event_listeners.js]: @AllanFly120 

How about this message instead:

> Missing credentials in config, if using AWS_CONFIG_FILE, set AWS_SDK_LOAD_CONFIG=1

Comment (odykyi) [reply lib/event_listeners.js]: @srchase sure!

**Comment by ajredniwja** [comment]
This change has already been introduced.

**Pull request closed without merging**